compiler/rustc_macros/src/symbols.rs

```diff
@@ -298,7 +298,7 @@ fn symbols_with_errors(input: TokenStream) -> (TokenStream, Vec<syn::Error>) {
     let preinterned_symbols_count = entries.len();
     let output = quote! {
         const SYMBOL_DIGITS_BASE: u32 = #symbol_digits_base;
-        const PREINTERNED_SYMBOLS_COUNT: u32 = #preinterned_symbols_count;
+        pub const PREINTERNED_SYMBOLS_COUNT: u32 = #preinterned_symbols_count;
```

Comment:
This needs a better name and we should document what it really means. Perhaps something like

```suggestion
        /// The number of predefined symbols; this is the the first index for
        /// extra pre-interned symbols in an Interner created via
        /// `new_with_extra_preinterned_symbols`.
        pub const PREDEFINED_SYMBOLS_COUNT: u32 = #predefined_symbols_count;
```